Scripts/Mobiles/BaseCreature.cs

```diff
@@ -5260,6 +5260,10 @@ public override void OnDeath(Container c)
 							givenToTKill = true;
 							TreasuresOfTokuno.HandleKill(this, ds.m_Mobile);
 						}
+                        if (Map == Map.Malas || region.IsPartOf("GauntletRegion"))

```

Comment:
This would be &&, not ||, correct? Otherwise you'll be calling this logic for every kill in Mala, which isn't necessary.